test/test_flask_v3.py

```diff
@@ -1,21 +1,28 @@
 import inspect
 import os
+from functools import wraps
 
 import mock
 import pytest
 
 from doctor.flask import (
     handle_http_v3, HTTP400Exception, should_raise_response_validation_errors)
 from doctor.response import Response
-from doctor.routing import get_params_from_func
-from doctor.types import integer, boolean, Object, new_type
+from doctor.utils import (
+    add_param_annotations, get_params_from_func, Params, RequestParamAnnotation)
 
+from .types import Auth, Item, ItemId, IncludeDeleted
 
-ItemId = integer('item id', minimum=1)
```

Comment:
Moved all these custom types into `test/types.py` so they could be reused in different test files w/o having to redeclare them over and over again.